stable/grafana/values.yaml

```diff
@@ -149,3 +149,44 @@ grafana.ini:
     mode: console
   grafana_net:
     url: https://grafana.net
+## LDAP Authentication can be enabled with the following values on grafana.ini
+## NOTE: Grafana will fail to start if the value for ldap.toml is invalid
+  # auth.ldap:
+  #   enabled: true
+  #   allow_sign_up: true
+  #   config_file: /etc/grafana/ldap.toml
+
+## Grafana's LDAP configuration
+## Templated by the template in _helpers.tpl
+## NOTE: To enable the grafana.ini must be configured with auth.ldap.enabled
+## ref: http://docs.grafana.org/installation/configuration/#auth-ldap
+## ref: http://docs.grafana.org/installation/ldap/#configuration
+ldap:
+  enabled: false
+  createConfig: true
+  name: ""
```

Comment:
I think it will be useful to add a comment saying that `name:` is only necessary if `createConfig: false`

Reply:
Maybe is better to replace `createConfig` for `createSecret` to be more explicit and use `secretName` instead of `name`

Reply:
updated